### DISCUSSION TITLE
最終ピッチたたき台

### DISCUSSION THREAD

**Discussion by tsmiyamoto** [YAKIONIGIRIs/Mapda3#16] [Ideas]
YouTubeをベースに、叩いていく

https://www.youtube.com/watch?v=dSb38hC0AXw

**Comment by tsmiyamoto** [comment]
1. 課題

**Reply by tsmiyamoto** [reply]
今日のドライブルート
- 家族や恋人と走ったあの道がうんたら
- YouTubeの映像、どこを走っているかわかんないよね


目的地まで
- Googleマップ・ナビで検索して出てくるのは幹線道路を走る_おもんない_ルート

今のナビやドラレコに「面白味はない」ということを主張

**Comment by tsmiyamoto** [comment]
2. Mapda3（概要文と共に）

**Comment by tsmiyamoto** [comment]
3. あなたの経路を自動でNFT化


※ 経路：緯度・経度の連続情報
※ 画像：その緯度・経度ごとに収集される画像
※ マップ：経路＋画像

**Comment by tsmiyamoto** [comment]
4. ストリートビュー付きの地図を作成

**Reply by tsmiyamoto** [reply]
自分から見た1方向？技術的にはアラウンドビューモニター的なんで円柱の側面みたいな形でストリートビュー作成は可能

**Comment by tsmiyamoto** [comment]
5. ストリートビューの作成を行きかう車と協力

**Comment by tsmiyamoto** [comment]
同乗者が写真を追加できる機能

**Reply by tsmiyamoto** [reply]
今後追加しうる機能として言うか言わないか（時間次第）

**Comment by tsmiyamoto** [comment]
6. 作成した地図を共有

個人がNFTマップを購入する動機は？（Entertainmentの文脈）
- 思い出として
  - あの日あの時の旅行というのを、経路とパノラマ画像で振り返れるようになる
  - 地図を所有するという今までにない形
- **旅行プランとしての販売**
- 写真を買うような感覚で地図を買える
- **旅をするインフルエンサーが「私たちの通った道」を売ることも可能**
  - ブラタモリ、ここ通ったんだ！（ブラタモリは歩きだけどw）

**Reply by tsmiyamoto** [reply]
「あの人、あの番組で使われたドライブルートを走りたい！」
- 日本列島ダーツの旅
- 出川哲郎の充電させてもらえませんか？

**Comment by tsmiyamoto** [comment]
7. 実用的なユースケースは？
サードパーソンビューなドラレコとしての事故検証

**Comment by tsmiyamoto** [comment]
8. 「経路」と「画像」が分散していた世界から、個々人が「自分の意志で」共有する時代に

MAZDAさん、色々使えるで～
（例えば？）

**Reply by tsmiyamoto** [reply]
その道の周辺で商売している第3次産業の人たち向けの情報販売ができる

1. Mapda3上に上がったNFTマップをマツダがClaim（この時点で個人に対しての報酬は払われる）
2. マツダが、それをもとにタイムズにおすすめ出店地教えるとか、色々なレポートを作成して販売することもできる（マツダの収益）
3. 三菱地所さん！貴社も不動産価値の再定義ができるかもよ！

**Reply by tsmiyamoto** [reply]
不動産価値の再定義ってなんだ？

**Reply by BB-Cotton** [reply]
その不動産の前の交通量とかは従来も分かってたけど、Mapda3が普及すればどこから来る車が多くて、どこに行くのかみたいな情報も簡単に手に入るようになるかも!?ってことだと理解してる。お金もかからないだろうし、利用規約の同意も得やすいだろうしね。

**Reply by BB-Cotton** [reply]
うまいことすれば、その不動産の前をよく通ってる人にピンポイントに広告出したりも出来そうね。

**Reply by tsmiyamoto** [reply]
Beautiful な回答や

**Comment by tsmiyamoto** [comment]
9. 動画
- 文字大きく？

**Comment by tsmiyamoto** [comment]
**To Do**
Due: 12/6(水)


- [x] スライド1～3 ( @toryo-bobo )
- [ ] スライド4～6 ( @BB-Cotton )
- [ ] スライド7～9 ( @tsmiyamoto )
- [ ] 360度ストリートビュー作成用の画像撮影（ @tsmiyamoto @toryo-bobo @BB-Cotton ）→よさげな画像を選定
